src/main/resources/application.yaml

```diff
@@ -48,6 +48,13 @@ folio:
       - name: links.instance-authority
         numPartitions: ${KAFKA_INSTANCE_AUTHORITY_TOPIC_PARTITIONS:10}
         replicationFactor: ${KAFKA_INSTANCE_AUTHORITY_TOPIC_REPLICATION_FACTOR:}
+    listener:
+      authority:
+        concurrency: ${KAFKA_AUTHORITIES_CONSUMER_CONCURRENCY:1}
+        topic-pattern: ${KAFKA_AUTHORITIES_CONSUMER_PATTERN:(${folio.environment}\.)(.*\.)inventory\.authority}
+        group-id: ${folio.environment}-mod-entities-links-authorities-group
+  retry:
+    enabled: true
```

Comment:
What is this parameter for?
Is it on a correct level (folio.retry), or should be related to kafka (folio.kafka.retry)?

Reply:
The retry mechanism that is implemented in folio-service-tools is not only for Kafka. It is for batch processing. And the property is to turn on beans creation for the retry mechanism. That's why it is on that level.

Reply:
Got it, thanks